corehq/apps/hqmedia/view_helpers.py

```diff
@@ -106,3 +110,22 @@ def update_multimedia_paths(app, paths):
                                  "{} > {}".format(module.default_name(), form.default_name())))
 
     return successes
+
+
+def download_audio_translator_files(domain, app, lang):
+    # Get bulk app translation single sheet data
+    sheets = get_bulk_app_single_sheet_by_name(app, lang, eligible_for_transifex_only=True)
```

Comment:
nit: it would be nice to let user decide if they want just transifex eligible translations.

Reply:
Good idea, added in fe7b38a0668 ...I'd love to delete that transifex blacklist code.